var/spack/repos/builtin/packages/py-mpi4py/package.py

```diff
@@ -37,6 +38,7 @@ class PyMpi4py(PythonPackage):
     depends_on("fortran", type="build")  # generated
```

Comment:
I'm not sure what this is about, but just in case: mpi4py does not depend on C++ or Fortran, the C compiler is enough.

Reply:
Maybe this line was autogenerated simply because the mpi4py tarball contains some C++ and Fortran code in the demo/ directory to showcase interoperability with other languages.

Reply:
Ok removed the C++, Fortran lines